src/foobar/migrations/0017_walletcorrection.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-02-17 15:40
+from __future__ import unicode_literals
+
+from decimal import Decimal
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import djmoney.models.fields
+import enumfields.fields
+import foobar.enums
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wallet', '0006_auto_20170217_1540'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('foobar', '0016_migrate_product_trx_references'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='WalletCorrection',
+            fields=[
+                ('date_created', models.DateTimeField(auto_now_add=True, null=True, verbose_name='date created')),
+                ('date_modified', models.DateTimeField(auto_now=True, null=True, verbose_name='date modified')),
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('trx_type', enumfields.fields.EnumIntegerField(default=0, enum=foobar.enums.TrxType)),
+                ('comment', models.CharField(blank=True, max_length=128, null=True)),
+                ('amount_currency', djmoney.models.fields.CurrencyField(choices=[('SEK', 'Swedish Krona')], default='SEK', editable=False, max_length=3)),
+                ('amount', djmoney.models.fields.MoneyField(decimal_places=2, default=Decimal('0.0'), max_digits=10)),
+                ('post_balance_currency', djmoney.models.fields.CurrencyField(choices=[('SEK', 'Swedish Krona')], default='SEK', editable=False, max_length=3)),
+                ('post_balance', djmoney.models.fields.MoneyField(decimal_places=2, default=Decimal('0.0'), max_digits=10)),
+                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('wallet', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='wallet.Wallet')),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+    ]
```

Comment:
What happens with old transactions that exist in the system?

I assume that each new transaction will generate a Correction/Log but should/can we generate an instance for the old ones?